src/error.rs

```diff
@@ -170,6 +173,11 @@ impl From<NetworkError> for Error {
                 message: format!("Failed to generate wpa passphrase for {}: {}", ssid, source),
                 data: None,
             },
+            NetworkError::GenWpaPassphraseWarning { ssid, err_msg } => Error {
+                code: ErrorCode::ServerError(-32027),
```

Comment:
The `ServerError` code (`-32027`) is already taken by `NetworkError::Connect`. I suggest changing this to `-32036` (the lowest available code). At some point I need to generate a simple list of these codes and their respective errors.

The [JSON RPC 2.0 spec](https://www.jsonrpc.org/specification#error_object) defines the `-32000` to `-32099` range for 'implementation-defined server-errors'. `peach-network` implements a unique code within that range of each of the different errors that can occur.

Reply:
aw good catch